docs/get-started/apps-integrations.md

```diff
@@ -34,22 +34,9 @@ Certain apps have specific installation requirements. Be sure to check the instr
 Applications can be installed only by users with a Sumo Logic Professional or a Sumo Logic Enterprise account. Organizations with a Sumo Logic Free account currently cannot use Sumo Logic Apps.
 :::
 
-To install an application:
+To install a Sumo Logic app:
```

Comment:
```suggestion

```

Reply:
You need to remove the line "To install a Sumo Logic app:" because the /reuse/apps/app-install.md file already starts with "To install the app:".